src/api.jl

```diff
@@ -36,6 +40,10 @@ new variables and constraints.
     relaxation with at most `n+m` partitions.
 - `variable_pre_base_name::AbstractString`: base_name that needs to be added to the auxiliary
     variables for meaningful LP files
+- `constraint_pre_base_name::AbstractString`: base_name that needs to be added to the constraints
+    in the relaxation.
+- `use_formulation_info::FormulationInfo` : `FormulationInfo` for another univariate function on the 
+    same variable that can be reused for the current relaxation. 
```

Comment:
Looks like the correct parameter name is `formulation_info` and not `use_formulation_info`.